sentinel-core/src/main/java/com/alibaba/csp/sentinel/config/SentinelConfig.java

```diff
@@ -98,6 +103,21 @@ private static void resolveAppType() {
         }
     }
 
+    private static void resolveMaxContextNameSize() {
+        try {
+            String size = getConfig(MAX_CONTEXT_NAME_SIZE);
+            if (size == null) {
+                return;
+            }
+            maxContextNameSize = Integer.parseInt(size);
+            if (maxContextNameSize < 0) {
+                maxContextNameSize = DEFAULT_MAX_CONTEXT_NAME_SIZE;
+            }
```

Comment:
Maybe log a warning message here to tell the user that we had overwritten the `maxContextNameSize` value

Reply:
I need a standard warning message